app/views/transactions/new.html.slim

```diff
@@ -0,0 +1,4 @@
+= render "partials/header"
```

Comment:
You don't actually need to render the header since its been imported in application.html.erb, meaning that by default it is included in every view file